.travis.yml

```diff
@@ -12,9 +12,9 @@ before_install:
 
 env:
   - rails=4.2.0
-  - rails=5.0.0.beta3
+  - rails=5.0.0.rc1
 
 matrix:
   exclude:
-    - env: rails=5.0.0.beta3
+    - env: rails=5.0.0.rc1
```

Comment:
I don't see why this appears as a diff if `master` contains this commit: https://github.com/doorkeeper-gem/doorkeeper/commit/eb40ba04518b76bc074865114e46e8bc9f549333.

Also, I look like the author of your rebased history. Do you want to `git commit --amend --reset-author`?